projects/components/karma.conf.js

```diff
@@ -21,6 +21,12 @@ module.exports = function (config) {
       fixWebpackSourcePaths: true
     },
     reporters: ['progress', 'kjhtml'],
+    customLaunchers: {
```

Comment:
does this work local on a dev pc with normal chrome installed?

Reply:
it does

Reply:
ok, good